### PR TITLE
feat: add all non-goerli Etherscan supported chains

### PR DESCRIPTION
**Pull request by antazoey**
### What I did

I basically need all chains listed here: https://docs.etherscan.io/contract-verification/supported-chains
to work (besides any goerli) so that is what this PR adds.

### Checklist

- [ ] All changes are completed
- [ ] New test cases have been added
- [ ] Documentation has been updated

### CODE REVIEW

File: evmchains/chains.py [hunk 1]
```diff
@@ -405,6 +459,142 @@
...
+            "icon": None,
+            "infoURL": "https://bt.io",
+            "name": "BitTorrent Chain Mainnet",
+            "nativeCurrency": {"decimals": 18, "name": "BitTorrent", "symbol": "BTT"},
+            "networkId": 199,
+            "rpc": ["https://rpc.bt.io", "https://bittorrent.drpc.org"],
+            "shortName": "BTT",
+            "slip44": None,
+        },
+        "donau": {
```
Comment (antazoey) [inline evmchains/chains.py:522]: etherscan calls this `donau` but i dont see it mentioned in the data so not sure about that

File: scripts/update.py [hunk 2]
```diff
@@ -47,11 +48,21 @@
     "blast": {
         "mainnet": 81457,
         "sepolia": 168587773,
+        "testnet": 23888,
     },
     "bsc": {
         "mainnet": 56,
         "testnet": 97,
         "opBNB": 204,
+        "opBNB-testnet": 5611,
```
Comment (mikeshultz) [inline scripts/update.py]: question: Are we sure we want these to be capitalized? I assume everything leading up to this point is case sensitive.

**Review by mikeshultz — APPROVED** [review]

File: scripts/update.py (hunk 2 above)
Comment (antazoey) [reply scripts/update.py]: ya lower is probably better

Comment (antazoey) [reply scripts/update.py]: done!

**Comment by antazoey** [comment]
about to change frax to fraxtal
edit: done

File: evmchains/chains.py (hunk 1 above)
Comment (antazoey) [reply evmchains/chains.py:522]: going to investigate and resolve this before merging

Comment (antazoey) [reply evmchains/chains.py:522]: yep, their testnet is called donau, made sure: https://doc.bt.io/docs/networks/network/#bttc-donau-testnet

**Pull request merged**